### DISCUSSION TITLE
Remove the duplicate button in collections with auth: true (like users)

### DISCUSSION THREAD

**Discussion by fabioquarantini** [payloadcms/payload#2429] [Feature Requests & Ideas]
If I click the duplicate button in a collection with "auth: true" it generates an error.
This is because the email is unique and cannot be duplicated.
Maybe the button needs to be hidden.

**Comment by jmikrut** [comment]
Hey there - you have a few options here!

First up, you can set `admin.disableDuplicate: true` which will remove the button.

But second, there is a `beforeDuplicate` hook that allows you to transform the data of the document being duplicated to modify the email address before the doc is created.

Docs are here:
https://payloadcms.com/docs/configuration/collections#admin-hooks

Whaddya think?

**Comment by fabioquarantini** [comment]
Hi @jmikrut,
thanks for the quick response.
Yes, I know the two options. My guess is for the default behavior that throws error.
